setup.py

```diff
@@ -56,6 +56,7 @@ def read_file(fname):
 install_requires = [
     'colorlog',
     'django>=1.9,<1.10',
+    'django-filter==0.13.0',
```

Comment:
Yeah, that would defeat the "pinning" -- despite the goal of backwards compatibility with semantic versioning.... doesn't always happen.